doc/develop/getting_started/index.rst

```diff
@@ -24,7 +24,7 @@ Click the operating system you are using.
 
    .. group-tab:: Ubuntu
 
-      This guide covers Ubuntu version 18.04 LTS and later.
```

Comment:
I think guide should be updated instead. Can you please open an issue, mentioning what doesn't work?

Reply:
Last time I checked, everything seemed to work on Ubuntu 22.04 LTS.

If there are any incompatibilities, we should update the guide as @gmaurll noted above.

Reply:
The problem occurs in [install dependencies step](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#install-dependencies), where the `kitware-archive.sh` script says it supports only `Only Ubuntu Xenial (16.04), Bionic (18.04), and Focal (20.04)` (https://apt.kitware.com/kitware-archive.sh).

Reply:
https://github.com/zephyrproject-rtos/sdk-ng/issues/464#issuecomment-1114579342

> However, note that the version of CMake provided by the distro is 3.22.1, which is greater than the minimum required version of 3.20.0; so, this step is not absolutely required.

This step is not required for Ubuntu 22.04, but I do note that there is no mention of such in the guide.

The Getting Started Guide should be updated such that the CMake installation is an optional step.